relay-server/src/actors/processor.rs

```diff
@@ -2266,7 +2266,8 @@ impl EnvelopeProcessorService {
             .and_then(|v| v.get("span.system"))
             .and_then(|system| system.as_str())
             .unwrap_or_default();
-        op == "http.client"
+        resource_span_extraction_enabled && op.contains("resource.")
```

Comment:
I'm never sure which operator takes precedence, so I would prefer parentheses here.
```suggestion
        (resource_span_extraction_enabled && op.contains("resource."))
```